src/Entity/Geo/ImpreciseGeoCoordinates.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Justimmo\Api\Entity\Geo;
+
+use Justimmo\Api\Annotation as JUSTIMMO;
+
+/**
+ * @JUSTIMMO\Entity()
+ */
+class ImpreciseGeoCoordinates extends GeoCoordinates
+{
+    public const TYPE_FUZZY   = 'fuzzy';
+    public const TYPE_PLZ_ORT = 'plz_ort';
+    public const TYPE_INVALID = 'invalid';
+
+    /**
+     * @var string
+     * @JUSTIMMO\Column(path="type", type="string")
+     */
+    private $type;
+
+    /**
+     * @var int
+     * @JUSTIMMO\Column(path="radius", type="integer")
+     */
+    private $radius;
+
+    public function getType() : string
```

Comment:
private constanten mit is*() : bool methoden wären eventuell sprechender anstatt ein allgemeiner getter